### PR TITLE
Adding Mermaid Chart Support

### PR DESCRIPTION
**Pull request by leogdion**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -95,7 +95,10 @@ analyze:
 db-up: db-up-dev db-up-test
 
 db-up-dev:
-	docker run --name spi_dev -e POSTGRES_DB=spi_dev -e POSTGRES_USER=spi_dev -e POSTGRES_PASSWORD=xxx -p 6432:5432 -d postgres:16-alpine
+	docker run --name spi_dev -e POSTGRES_DB=spi_dev -e POSTGRES_USER=spi_dev -e POSTGRES_PASSWORD=xxx -p 6432:5432 \
+	-e PGDATA=/pgdata \
+	--tmpfs /pgdata:rw,noexec,nosuid,size=1024m \
+	-d postgres:16-alpine
```
Comment (finestructure) [inline Makefile:101]: We want the local `dev` db to persist changes between runs (as opposed to the local `test` db that's used to run unit tests). That's why it's not using the `--tmpfs` flag!

**Comment by daveverwer** [comment]
Thanks @leogdion. I just sent you over a CLA and I'll take a look at this in detail this week. 👍

File: Makefile (hunk 1 above)
Comment (leogdion) [reply Makefile]: Change reverted. Just for reference. Here's why:
https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/3348#issuecomment-2435945571

**Comment by leogdion** [comment]
Thank you! CLA is signed. Please let me know if you need anything else.

**Comment by daveverwer** [comment]
@cla-bot recheck

**Comment by daveverwer** [comment]
@cla-bot check

**Comment by daveverwer** [comment]
Superceded by #3499

**Pull request closed without merging**